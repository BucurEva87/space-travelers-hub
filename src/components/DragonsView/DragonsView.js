import React from 'react';
import Dragon from '../Dragon/Dragon';
// import style from './DragonsView.module.scss';

const DragonsView = () => (
  <Dragon />
);

export default DragonsView;
