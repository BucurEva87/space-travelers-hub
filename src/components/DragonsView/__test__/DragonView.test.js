import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../../../redux/configureStore';
import DragonsView from '../DragonsView';

let getByTestId;

beforeEach(() => {
  const component = render(
    <Provider store={store}>
      <DragonsView />
    </Provider>,
  );

  getByTestId = component.getByTestId;
});

describe('DragonsView component', () => {
  test('renders on the screen', () => {
    expect(getByTestId('dragons-container')).toBeInTheDocument();
  });

  test('matches snapshot', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <DragonsView />
      </Provider>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
