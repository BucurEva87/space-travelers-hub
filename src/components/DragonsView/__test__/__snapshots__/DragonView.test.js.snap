// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DragonsView component matches snapshot 1`] = `
<section
  data-testid="dragons-container"
/>
`;
